src/Test/Utilities/Portable/Mocks/TestStream.cs

```diff
@@ -60,6 +61,10 @@ public override long Position
                 {
                     return _backingStream.Position;
                 }
+                if (_backingStream != null)
+                {
+                    return _backingStream.Position;
+                }
```

Comment:
📝 This is another bug in the batch fixer or in the workspaces layer. This block is a duplicate of the one right before it.

Reply:
thats's significantly worrying.  i would be surprised if it was in the workspace.  it wouldn't surprise me if it was in the batch fixer.

Reply:
My guess right now is the workspace code that merges changes that occur in linked files. Prior to writing the changes to disk, the linked files act like multiple independent copies. When the changes are applied, they all have to go to the same file on disk.